ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -37879,6 +37879,111 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: '@yearly'
+  decorate: true
+  decoration_config:
+    timeout: 6h0m0s
+  name: periodic-ci-openshift-release-master-nightly-4.12-install-analysis-gcp-ovn-ipi
+  spec:
+    containers:
+    - args:
+      - --gcs-upload-secret=/secrets/gcs/service-account.json
+      - --give-pr-author-access-to-namespace=true
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      - --lease-server-credentials-file=/etc/boskos/credentials
+      - --report-credentials-file=/etc/report/credentials
+      - --secret-dir=/usr/local/pull-secret
+      - --target=release-analysis-install-analysis
+      command:
+      - ci-operator
+      env:
+      - name: GOOGLE_SA_CREDENTIAL_FILE
+        value: /var/run/secrets/google-serviceaccount-credentials.json
+      - name: INFRASTRUCTURE
+        value: ipi
+      - name: JOB_START_TIME
+      - name: MINIMUM_SUCCESSFUL_COUNT
+        value: "2"
+      - name: NETWORK
+        value: ovn
+      - name: PAYLOAD_TAG
+      - name: PLATFORM
+        value: gcp
+      - name: TEST_GROUP
+        value: install
+      - name: UNRESOLVED_CONFIG
+        value: |
+          resources:
+            '*':
+              limits:
+                memory: 6Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tests:
+          - as: release-analysis-install-analysis
```

Comment:
Did you check with test platform to see if moving this to config folder will cause side effect for "make jobs"? Ideally we can do that in stead of duplicating this section for each job.

Reply:
I will do that.  But first let me see if I can figure out where it will go in the 'config' subdir.